Tests/MSFT_xFirewall.Tests.ps1

```diff
@@ -0,0 +1,15 @@
+Remove-Module -Name MSFT_xFirewall -Force -ErrorAction SilentlyContinue
+Import-Module -Name $PSScriptRoot\..\DSCResources\MSFT_xFirewall -Force -DisableNameChecking
+
+if (! (Get-Module xDSCResourceDesigner))
+{
+    Import-Module -Name xDSCResourceDesigner -ErrorAction SilentlyContinue
+}
+
+Describe 'Schema Validation for MSFT_xFirewall' {
+    Copy-Item -Path ((get-item .).parent.FullName) -Destination $(Join-Path -Path $env:ProgramFiles -ChildPath 'WindowsPowerShell\Modules\') -Force -Recurse
+    $result = Test-xDscResource MSFT_xFirewall
```

Comment:
If the Test-xDscResource line it self fails, pester will tell you a more specific failure context if you include that line in the it.  If it's not specifically related to the IT, then it should stay in the describe.